### ISSUE TITLE
DSK/NIB: All even disk i/o read accesses will read the data latch

### ISSUE THREAD

**Issue by tomcw** [AppleWin/AppleWin#673]
Like #672, but for .dsk and .nib images.

@Archange427 said:

>Not strictly related to this WOZ reading problem, but I remember that some groups of the french underground cracking scene in the 80s loved to use "exotic" addresses for their reading routine (like $C0E0 instead of $C0EC).
When I was doing disk preservation, I had to patch a lot of these 'fresh' .DSK to work on AppleWin.
Could you also fix this issue for DSK ?
I can send you some DSK for tests if you want!

Yes - please attach (or send) DSK images for testing.
Thanks.

**Comment by Archange427** [comment]
Here is an assortment! 
There was probably more but at the time, I was directly patches DSK that I did so they boot without thinking to keep the original files. I was stupid...
And now the floppies are lost...
[DSK not booting (C080).zip](https://github.com/AppleWin/AppleWin/files/3458207/DSK.not.booting.C080.zip)

**Comment by sicklittlemonkey** [comment]
Yes, because (e.g.) $C0E0 reads the latch on AppleWin but doesn't advance the stream position - unless it's a WOZ now.
